components/admin/other-members.tsx

```diff
@@ -48,6 +60,27 @@ export default function OtherMembers({
                       `(age ${getAgeInYears(applicant)})`}
                   </li>
                 </ul>
+
+                <button
+                  onClick={() => handleDeleteDialog(applicant)}
+                  className="lbh-body-s lbh-link lbh-link--no-visited-state lbh-delete-link"
+                >
+                  Delete household member
+                </button>
+
+                <Dialog
+                  isOpen={showDeleteWarningDialog}
+                  title="Confirm delete"
+                  onConfirmation={() => handleDelete(applicantToDelete)}
+                  onCancel={() => setShowDeleteWarningDialog(false)}
+                >
+                  <Paragraph>
+                    Are you sure you want to delete{' '}
```

Comment:
Can I check in on that wording "delete"? I'm no UXer, but I think maybe "remove" might be better?

Reply:
No worries, I'll change it to remove!